src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -57,9 +58,12 @@ def _create_session(self, alias, url, headers, cookies, auth,
         s = session = requests.Session()
         s.headers.update(headers)
         s.auth = auth if auth else s.auth
-        s.proxies = proxies if proxies else  s.proxies
-
         s.verify = self.builtin.convert_to_boolean(verify)
+        if proxies:
+            s.proxies = ast.literal_eval(proxies)
```

Comment:
Well; the [Zen of Python](https://www.python.org/dev/peps/pep-0020/) writes;

>   Explicit is better than implicit.
 
That was the philosophy behind the approach I took to writing test cases as well.